### PR TITLE
[Translation] Handle the blank-translation in Loco Adapter

### LINKED ISSUE #44595: [Translation][Loco] Consider handling the special status "blank translation"
### Description

Currently, the adapter only handles the status "translated" when pulling translations. In loco, there is a special status "blank translation" which allows to say that this key is translated but empty. This status is useful because you cannot save an empty key with the status "translated". The export API allows you to combine the statuses if you separate them with "," like in the example

### Example

The method read of LocoProvider will look like this : 
```
$response = $this->client->request('GET', sprintf('export/locale/%s.xlf', rawurlencode($locale)), [
    'query' => [
        'filter' => $domain,
        'status' => 'translated, blank-translation',
    ],
]);
```

### PR DESCRIPTION
**Pull request by kgonella**
| Q             | A
| ------------- | ---
| Branch?       | 5.3, 5.4, 6.0
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #44595
| License       | MIT

This PR aim to be able to pull the "blank-translation" in addition to "translated" status translations. See the ticket to know more about. I don't think we can add a unit test here as the calls to loco are fully mocked.

### CODE REVIEW

File: src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public function read(array $domains, array $locales): TranslatorBag
                 $response = $this->client->request('GET', sprintf('export/locale/%s.xlf', rawurlencode($locale)), [
                     'query' => [
                         'filter' => $domain,
-                        'status' => 'translated',
+                        'status' => 'translated, blank-translation',
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php]: shouldn't we remove the space?

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php (hunk 1 above)
Comment (kgonella) [reply src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php]: Both work, it's up to you

Comment (nicolas-grekas) [reply src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php]: please remove it. A space in a query string looks odd

Comment (kgonella) [reply src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php]: Done

**Review by welcoMattic — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @kgonella.

**Pull request merged**